io.embrace.sdk/CHANGELOG.md

```diff
@@ -5,9 +5,11 @@ sidebar_position: 4
 ---
 
 # Unity SDK Changelog
-## 2.0.3
-*DATE*
-* Upgrade of Embrace iOS SDK to 6.5.1
+## 2.1
+*November 21, 2024*
+* Upgrade of Embrace Android dependency to 6.14.0
```

Comment:
was this in the previous PR? I may have missed that

Reply:
looks like it was in PR #39